tests/Jobs/UpdateWikiEntitiesCountJobTest.php

```diff
@@ -0,0 +1,191 @@
+<?php
+
+namespace Tests\Jobs;
+
+use App\Jobs\UpdateWikiEntitiesCountJob;
+use App\Wiki;
+use Illuminate\Contracts\Queue\Job;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Illuminate\Support\Facades\Bus;
+use Illuminate\Support\Facades\Http;
+use PHPUnit\Framework\MockObject\Exception;
+use Tests\TestCase;
+
+class UpdateWikiEntitiesCountJobTest extends TestCase
+{
+    use RefreshDatabase;
+
+    public function setUp(): void {
+        parent::setUp();
```

Comment:
Nitpick: If there is no setup that is specific to `UpdateWikiEntitiesCountJobTest` to be done, there is no need to redeclare `setUp`  like this. Same goes for `tearDown`